### PR TITLE
Fix target browser implementation. Plus updated the readme

### PR DESCRIPTION
**Pull request by EWhite613**
#minor#
Fixes https://github.com/ciena-blueplanet/ember-cli-visual-acceptance/issues/12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,22 +6,33 @@ Create baseline images and test for CSS regression during standard Ember tests u
...
+```javascript
 visualAcceptanceOptions: {
       imageDirectory: 'visual-acceptance'
     }
 ``` 
+* You can specify browsers to target by adding an object inside the `targetBrowsers` array in the `visualAcceptanceOptions` of `ember-cli-build.js`
+```javascript
+visualAcceptanceOptions: {
+      targetBrowsers: [{
+        bit: undefined,
```
Comment (sglanzer) [inline README.md]: Would it make sense to chop this down to just `browser` and `os` as mandatory?  I'd probably use `>=` for the `osversion` and `version` and I'm not sure what `bit` and `mobile` are supposed to control

File: index.js [hunk 2]
```diff
@@ -20,10 +20,14 @@ function mkdirpSync (dirpath) {
 }
 
 function isTargetBrowser (req, res, targetBrowsers) {
-  console.log(targetBrowsers)
   if (targetBrowsers.length > 0) {
-    var clientBrowser = JSON.stringify(req.body.browser)
-    var result = targetBrowsers.indexOf(clientBrowser) !== -1
+    var result = false
+    for (var i = 0; i < targetBrowsers.length; i++) {
+      if (req.query.browser === targetBrowsers[i].browser && req.query.version === targetBrowsers[i].version && req.query.os === targetBrowsers[i].os && req.query.osversion === targetBrowsers[i].osversion && (req.query.mobile === targetBrowsers[i].mobile || (req.query.mobile === '' && targetBrowsers[i].mobile === undefined)) && (req.query.bit === targetBrowsers[i].bit || (req.query.bit === '' && targetBrowsers[i].bit === undefined))) {
```
Comment (sglanzer) [inline index.js]: This is where I would use >= for the versions (and make them optional)

Comment (sglanzer) [inline index.js]: Also, line-wraps are your friend ;)

**Comment by EWhite613** [comment]
@sglanzer Made those comparison changes. version and osversion are now optional. And if osversion or version begin with `>=` we look if the versions are >= the target. Otherwise its just an equality check

**Comment by EWhite613** [comment]
@sglanzer Also added this feature https://github.com/ciena-blueplanet/ember-cli-visual-acceptance/issues/6 . Available by using `ember new-baseline`

**Comment by sglanzer** [comment]
Approved

**Pull request merged**